tldw_Server_API/app/core/DB_Management/Media_DB_v2.py

```
@@ -1254,6 +1289,165 @@ def execute_many(
+            "uuid": new_uuid,
+            "created_at": now,
+            "last_modified": now,
+            "version": 1,
+            "client_id": self.client_id,
+            "deleted": 0,
+            "prev_version": None,
+            "merge_parent_uuid": None,
+        }
+        placeholders = ", ".join([f":{k}" for k in data.keys()])
+        columns = ", ".join(data.keys())
+        sql = f"INSERT INTO VisualDocuments ({columns}) VALUES ({placeholders})"
```

Ruff (0.14.6)

Possible SQL injection vector through string-based query construction

(S608)